src/amber/router/router.cr

```diff
@@ -29,9 +29,9 @@ module Amber
       end
 
       def draw(valve : Symbol, namespace : String)
+        scope = Scope.new
```

Comment:
Can we have a separate PR for the scope fix makes it easier to review and manage

Reply:
The scope is already created line `15`

https://github.com/amberframework/amber/pull/1121/files#diff-d4d79b3524306c32cadbb606d98afc53R15